microsoft-edge/toc.yml

```diff
@@ -1,327 +1,550 @@
+          items:
+            - name: Track changes to files using the Changes tool # added article
               href: devtools-guide-chromium/changes/changes-tool.md
-            - name: Console
-              items:
-                - name: Console overview
-                  href: devtools-guide-chromium/console/index.md
-                - name: Log messages
-                  href: devtools-guide-chromium/console/console-log.md
-                - name: Run JavaScript in the Console
-                  href: devtools-guide-chromium/console/console-javascript.md
-                - name: Console features reference
-                  href: devtools-guide-chromium/console/reference.md
-                - name: Console API reference
-                  href: devtools-guide-chromium/console/api.md
-                - name: Console Utilities API reference
-                  href: devtools-guide-chromium/console/utilities.md
-                - name: Watch JavaScript expressions in real time
-                  href: devtools-guide-chromium/console/live-expressions.md
-                - name: Fix JavaScript Console errors
-                  href: devtools-guide-chromium/console/console-debug-javascript.md
-                - name: Interact with the DOM
-                  href: devtools-guide-chromium/console/console-dom-interaction.md
-                - name: Filter Console messages
-                  href: devtools-guide-chromium/console/console-filters.md
-            - name: Coverage
+              displayName: Changes tool
+# Coverage tool ---------------------------------------------------------------
+        - name: Coverage tool
+          items:
+            - name: Find unused JavaScript and CSS code with the Coverage tool
               href: devtools-guide-chromium/coverage/index.md
-#           - name: Developer Resources
-            - name: Elements
-              items:
-                - name: CSS features reference
-                  href: devtools-guide-chromium/css/reference.md
-                - name: Edit CSS font styles and settings
-                  href: devtools-guide-chromium/inspect-styles/edit-fonts.md
-                - name: Inspect CSS grid
-                  href: devtools-guide-chromium/css/grid.md
-                - name: CSS-in-JS
-                  href: devtools-guide-chromium/css/css-in-js.md
-                - name: Force DevTools into Print Preview mode
-                  href: devtools-guide-chromium/css/print-preview.md
-            - name: Inspect
-              href: devtools-guide-chromium/css/inspect.md
-            - name: Issues
+              displayName: Coverage tool
+# CSS Overview tool (default tab 11) ------------------------------------------
+        # - name: CSS Overview tool
+        #   items:
+        #     - name: Identify potential CSS improvements # pending
+        #       href: devtools-guide-chromium/issues/index.md # pending
+        #       displayName: CSS Overview tool
```

Comment:
Maybe add a comment for these commented-out TOC nodes that explains why they're commented-out. It will help future us know what to do.